manifests/07-downloads-deployment.yaml

```diff
@@ -104,16 +89,11 @@ spec:
                   tar.add(path, basename)
               with zipfile.ZipFile('{}.zip'.format(archive_path_root), 'w') as zip:
                   zip.write(path, basename)
-
-          # Create socket
+          
           addr = ('', 8080)
-          sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-          sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-          sock.bind(addr)
-          sock.listen(5)
-
-          [Thread(i, socket=sock) for i in range(100)]
-          time.sleep(9e9)
+          server = ThreadedHTTPServer(addr, SimpleHTTPServer.SimpleHTTPRequestHandler)
+          print('Starting server, use <Ctrl-C> to stop')
```

Comment:
I don't think we need this line; nobody's at a teminal to ctrl-c it.

Reply:
Fixed